monkey/monkey_island/windows/create_certificate.bat

```diff
@@ -20,13 +20,13 @@ copy "%mydir%windows\openssl.cfg" "%mydir%bin\openssl\openssl.cfg"
 
 :: Change file permissions
 SET adminsIdentity="BUILTIN\Administrators"
-FOR /f %%O IN ('whoami') DO SET ownIdentity=%%O
+FOR /f "delims=''" %%O IN ('whoami') DO SET ownIdentity=%%O
 
 FOR %%F IN ("%mydir%cc\server.key", "%mydir%cc\server.csr", "%mydir%cc\server.crt") DO (
 
 	:: Remove all others and add admins rule (with full control)
-	echo y| cacls %%F" /p %adminsIdentity%:F
+	echo y| cacls %%F /p %adminsIdentity%:F
 
 	:: Add user rule (with read)
-	echo y| cacls %%F /e /p %ownIdentity%:R
+	echo y| cacls %%F /e /p "%ownIdentity%":R
```

Comment:
If we don't need the quotes around `%adminsIdentity%` above, do we need them here?

Reply:
Yes we do because  space can occur in user name. So with `FOR /f "delims=''" %%O IN ('whoami') DO SET ownIdentity=%%O` we are getting the full name of a user but because of space the command is not working, so that's why it is wrapped. `%adminsIdentity%` is a constant.